app/Models/Competition.php

```diff
@@ -31,16 +31,19 @@ public function users()
     /**
      * Gets a CSV export of all the users in this competition.
      *
+     * @param \Gladiator\Repositories\UserRepositoryContract $repository
      * @return \League\Csv\ $csv
      */
-    public function getCSVExport()
+    public function getCSVExport($repository)
```

Comment:
feels odd to take in the `$repository` here. Shouldn't it just pass data? If you need the `$repository` shouldn't you instantiate it in the Model instead? @weerd thoughts on this?